### DISCUSSION TITLE
Split tunnel a whole folder for example Games

### DISCUSSION THREAD

**Discussion by TheRealDadbeard** [mullvad/mullvadvpn-app#8956] [Feature requests & Ideas]
I add my Games to split tunneling as I want to have the lowest ping possible and no disconnection that already happened a couple of times but this is tiresome and Xbox Games don't let you select the exe files even if you can now see the whole folder.

Would it be possible to just whitelist a whole folder?

**Comment by faern** [comment]
Thanks for the suggestion. This would probably require significant changes to our split tunneling implementation. But we can keep it in mind and see.

**Comment by TheRealDadbeard** [comment]
Would be really cool to see this implemented.

**Comment by Turtlecute33** [comment]
UP, it's a pain right now and other VPN apps have implemented this (see, for example, Wiresock).

It would be really nice to finally have something like that on the Mullvad app too.
